Lib/fontTools/otlLib/builder.py

```diff
@@ -544,6 +544,10 @@ def buildFormat2Subtable(self, ruleset, classdefs, chaining=True):
                 f"{classRuleAttr}Count",
                 getattr(setForThisRule, f"{classRuleAttr}Count") + 1,
             )
+        for i, classSet in enumerate(classSets):
+            if not getattr(classSet, classRuleAttr):
+                # we can drop empty class sets that have no rules
```

Comment:
I might rephrase to something like "class sets can be null so replace nop sets with None"?

Reply:
thanks, I changed to your wording